### PR TITLE
Add UTexScaler, Fix ULevel and UModel signatures

### PR DESCRIPTION
**Pull request by metallicafan212**

### CODE REVIEW

File: Engine/Inc/UnLevel.h [hunk 1]
```diff
@@ -154,7 +154,8 @@ class ENGINE_API ULevel : public ULevelBase
 	enum {NUM_LEVEL_TEXT_BLOCKS=16};
 
 	// Main variables, always valid.
-	TArray<UReachSpec>		ReachSpecs;
+	// Not in UE2
+	//TArray<UReachSpec>		ReachSpecs;
```
Comment (Leon280698) [inline Engine/Inc/UnLevel.h]: Please remove if it is not needed

File: Engine/Inc/UnModel.h [hunk 2]
```diff
@@ -98,7 +98,9 @@ class ENGINE_API UModel : public UPrimitive
 	// UModel collision functions.
 	typedef void (*PLANE_FILTER_CALLBACK )(UModel *Model, INT iNode, int Param);
 	typedef void (*SPHERE_FILTER_CALLBACK)(UModel *Model, INT iNode, int IsBack, int Outside, int Param);
-	FPointRegion PointRegion( AZoneInfo* Zone, FVector Location ) const;
+	// Signature changed as it wouldn't link to Engine.dll
+	//FPointRegion PointRegion( AZoneInfo* Zone, FVector Location ) const;
```
Comment (Leon280698) [inline Engine/Inc/UnModel.h]: Remove irrelevant comments. Doing this for every function that needs adjustment just unnecessarily clutters the code

File: Core/Inc/Core.h [hunk 3]
```diff
@@ -20,7 +20,12 @@
 
 //===========================================================================================
 //! @brief Used to automatically link the proper lib for the dlls whose headers are currently included
+// Give an option to disable this, it can cause link errors if the libs are already defined in the project
```
Comment (Leon280698) [inline Core/Inc/Core.h]: Please move the documentation comment so that doxygen is able to generate documentation for the LINK_LIB macro

Comment (metallicafan212) [reply Core/Inc/Core.h]: Moved the doxygen comment next to the actual macro

File: Engine/Inc/UnModel.h (hunk 2 above)
Comment (metallicafan212) [reply Engine/Inc/UnModel.h]: Removed the comments

File: Engine/Inc/UnLevel.h (hunk 1 above)
Comment (metallicafan212) [reply Engine/Inc/UnLevel.h]: Removed the comments

**Comment by metallicafan212** [comment]
Fixed all the problems. I've never used Doxygen, so correct my placement if it's wrong.

**Pull request merged**